lib/conduit_async_ssl.ml

```diff
@@ -50,4 +50,9 @@ let ssl_listen ~crt_file ~key_file rd wr =
   >>= fun app_rd ->
```

Comment:
@rgrinberg I don't know either. I am eager to hear from a core committer about this.